tests/unit/amo/components/TestHeroRecommendation.js

```diff
@@ -153,34 +161,26 @@ describe(__filename, () => {
 
     it('configures an external link to open in a new tab', () => {
       const _checkInternalURL = sinon.stub().returns({ isInternal: false });
-      const external = fakePrimaryHeroShelfExternal;
+      const external = fakePrimaryHeroShelfExternalAddon;
       const shelfData = createShelfData({ external });
 
       const root = render({ _checkInternalURL, shelfData });
 
       const link = root.find('.HeroRecommendation-link');
       expect(link).toHaveProp('rel', 'noopener noreferrer');
       expect(link).toHaveProp('target', '_blank');
-      sinon.assert.calledWith(
-        _checkInternalURL,
-        sinon.match({ urlString: sinon.match(external.homepage) }),
-      );
```

Comment:
mmm, we don't need this check any longer?

Reply:
We don't strictly need it, but I suppose it makes the test safer, in case for some reason the mock wasn't called and yet the link was assumed to be external, so I'll add it back in. Thanks!